src/macros.rs

```diff
@@ -36,3 +36,38 @@ macro_rules! ordermap {
         }
     };
 }
+
+#[macro_export]
+/// Create an `OrderSet` from a list of values
+///
+/// ## Example
+///
+/// ```
+/// #[macro_use] extern crate ordermap;
+/// # fn main() {
+///
+/// let set = orderset!{
+///     "a",
+///     "b",
+/// };
+/// assert!(set.contains("a"));
+/// assert!(set.contains("b"));
+/// assert!(!set.contains("c"));
+///
+/// // "a" is the first value
+/// assert_eq!(set.iter().next(), Some(&"a"));
+/// # }
+/// ```
+macro_rules! orderset {
+    ($($value:expr,)+) => { orderset!($($value),+) };
+    ($($value:expr),*) => {
+        {
+            let _cap = ordermap!(@count $($value),*);
```

Comment:
I'd copy the count part of the macro to this place. It's the only way to support #[macro_use(orderset)] which I'd like to do whenever possible.

Reply:
Done.